File: problemas/ad-hoc/1546/1546.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');

const nomes = ["Rolien", "Naej", "Elehcim", "Odranoel"];

let N = lines.shift();

for(let i = 0; i < N; ++i){
  let K = lines.shift();

  for (let j = 0; j < K; ++j) {
    let feedback = lines.shift();

    console.log(nomes[feedback - 1]);
  }
}

File: problemas/ad-hoc/1546/1546.py
nomes = ['Rolien', 'Naej', 'Elehcim', 'Odranoel']

N = int(input())

for _ in range(N):
    K = int(input())

    for _ in range(K):
        feedback = int(input())

        print(nomes[feedback - 1])